PowerEditor/installer/nsisInclude/uninstall.nsh

```diff
@@ -271,8 +271,8 @@ Section Uninstall
 		Delete "$APPDATA\${APPNAME}\userDefineLangs\userDefinedLang-markdown.default.modern.xml"
 		RMDir /r "$APPDATA\${APPNAME}\plugins\"
 		RMDir /r "$APPDATA\${APPNAME}\backup\"	; Remove backup folder recursively if not empty
-		RMDir "$APPDATA\${APPNAME}\themes\"	; has no effect as not empty at this momenet, but it is taken care at un.onUninstSuccess
-		RMDir "$APPDATA\${APPNAME}"		; has no effect as not empty at this momenet, but it is taken care at un.onUninstSuccess
+		RMDir "$APPDATA\${APPNAME}\themes\"	; has no effect as not empty at this moment, but it is taken care at un.onUninstSuccess
+		RMDir "$APPDATA\${APPNAME}"		; has no effect as not empty at this moment, but it is taken care at un.onUninstSuccess
```

Comment:
@SinghRajenM:  If you are going to correct the spelling of a comment, you should also correct the *wording* of a comment. 

>it is taken care at 

…should be… 

>it is **dealt with** at 

"Taken care" is not a valid English phrase.  "Taken care of" is the correct form of the phrase, but it would be awkward in this particular sentence.  ("it is **taken care of** at un.onUninstSuccess")